tests/adobe/test_client.py

```diff
@@ -2800,23 +2915,9 @@ def test_get_flex_discounts(
         ],
     )
 
-    assert client._get_flex_discounts(
-        authorization_uk, "COM", "US", ("65304768CA01A12", "65304768CA01A12")
-    ) == [
-        {
-            "id": "55555555-8768-4e8a-9a2f-fb6a6b08f563",
-            "name": "Easter Flexible Discount",
-            "description": "Exclusive 26 fixed off on Adobe Technical Communication Suite",
-            "code": "EASTER_26",
-            "startDate": "2025-06-12T10:00:48Z",
-            "endDate": "2026-03-07T10:16Z",
-            "status": "ACTIVE",
-            "qualification": {"baseOfferIds": ["65304768CA01A12"]},
-            "outcomes": [
-                {
-                    "type": "FIXED_DISCOUNT",
-                    "discountValues": [{"country": "US", "currency": "USD", "value": 26.0}],
-                }
-            ],
-        }
-    ]
+    assert (
+        client._get_flex_discounts(
+            authorization, "COM", "US", ("65304768CA01A12", "65304768CA01A12")
+        )
+        == flex_discount_resp["flexDiscounts"]
```

Comment:
Why are we testing a private method? As far as I know, we have the ruff / wemake rule not to do that